python/pyspark/testing/utils.py

```diff
@@ -103,7 +103,7 @@ def close(self):
         pass
 
 
-def search_jar(project_relative_path, jar_name_prefix):
+def search_jar(project_relative_path, sbt_jar_name_prefix, mvn_jar_name_prefix):
```

Comment:
Hi, @HyukjinKwon . Since SPARK-26856 used this function for `spark-avro_2.12-3.0.0-SNAPSHOT` which is identical for both `sbt` and `mvn`. Inevitably, I split the parameters. How do you think about this approach?

Reply:
Yes, I can't think of another minimised fix. It's a testing util anyway. we can fix it later if we can come up with a better idea. LGTM.

While we're here, mind adding some comments? It doesn't need to be a proper docstring. We can just write something like:

```python
def search_jar(project_relative_path, sbt_jar_name_prefix, mvn_jar_name_prefix):
    # Note that 'sbt_jar_name_prefix' and 'mvn_jar_name_prefix' are separate since the prefix can
    # be varied for SBT or Maven specifically. See also SPARK-26856
    ...
```

Reply:
Thanks. No problem. It's added.